src/index.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-len */
+/* eslint-disable no-useless-concat */
+/* eslint-disable no-use-before-define */
+/* eslint-disable radix */
+/* eslint-disable no-undef */
+const appId = '70485a8e13b31fcaee6010aadf0328be';
+const units = 'metric';
+let searchMethod;
+
+const getSearchMethod = searchTerm => {
+  if (searchTerm.length === 5 && `${Number.parseInt(searchTerm)}` === searchTerm) {
+    searchMethod = 'zip';
+  } else searchMethod = 'q';
+};
+
+const searchWeather = searchTerm => {
+  getSearchMethod(searchTerm);
+  fetch(`https://api.openweathermap.org/data/2.5/weather?${searchMethod}=${searchTerm}&APPID=${appId}&units=${units}`).then(result => result.json()).then(result => { init(result); });
+};
```

Comment:
As it said in the requirements, error handling needs to be added to the fetch data, in case something went wrong.
![Screenshot from 2020-04-03 07-52-17](https://user-images.githubusercontent.com/30977845/78353215-119d3300-7580-11ea-9658-9efd5e6ff7b7.png)